server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -297,7 +296,7 @@ private static boolean isValidColumn(ColumnUpdate cu) {
   }
 
   protected Arbitrator getArbitrator() {
-    return new ZooArbitrator();
+    return new ZooArbitrator(ServerContext.getInstance());
```

Comment:
I think it would be good to open a follow on issue to explore ways to remove this.  There was a lot to look at so I may have missed some, but this is the only call I noticed to ServerContext.getInstance() that was not in a main method.

Reply:
I am not going to close #582 until all calls to getInstance() outside of main methods are removed and calls to getInstance() are converted to constructors.